BuildAndTest.sh

```diff
@@ -6,6 +6,11 @@ if [[  "$(uname)" == "Linux" || "$(uname)" == "Darwin" ]]; then
   sed 's#\\#/#g' src/BinSkim.sln > src/BinSkimUnix.sln
 fi
 
+if [ ! -f src/sarif-sdk/src/Sarif.Sdk.sln ]; then
+  echo "Get submodule..."
+  git submodule update --init --recursive
+fi
+
```

Comment:
looks good.
just throwing a open question if this new added code need to be inside of the 
`if [[  "$(uname)" == "Linux" || "$(uname)" == "Darwin" ]]; then .... fi`
code block or outside.

Reply:
@LingZhou-gh Just to check, please test the changes on a fresh enlistment from this branch.